### ISSUE TITLE
Overweight and Underweight a measurement?

### ISSUE THREAD

**Issue by PRijnbeek** [OHDSI/Vocabulary-v5.0#133]
Vocabulary Version 2017-08-04

Can someone explain me why overweight and underweight are in the measurement domain only?

To me this is an observation since it has no value. If this is not the difference between measurements and observation I am missing something.

Related issues: #130, #129

**Comment by aostropolets** [comment]
Both of them have observation domain now when belong to SNOMED.

**Comment by ericaVoss** [comment]
I say this issue is closed as it is fixed in 20170920.

```{SQL}
SELECT CONCEPT_ID, CONCEPT_NAME, DOMAIN_ID
FROM CONCEPT
WHERE CONCEPT_ID IN (
437525 /*Overweight*/, 
435372 /*Underweight*/
)
```

```
CONCEPT_ID	CONCEPT_NAME	DOMAIN_ID
435372	Underweight	Observation
437525	Overweight	Observation
```

**Comment by ericaVoss** [comment]
@PRijnbeek or @aostropolets could you please close this issue as it is fixed?  Still fixed in 20171201.
